src/Apps/Order/Routes/__tests__/Review.test.tsx

```diff
@@ -111,6 +115,30 @@ describe("Review", () => {
       )
     })
 
+    it("shows a modal with a helpful error message if the user's card is declined", async () => {
+      window.location.assign = jest.fn()
```

Comment:
Do we need this mock? (and same Q for test below?)

Reply:
@lilyfromseattle @ds300 popping in to ask a quick question. I don't have enough context to know whether this needs to be mocked but I was wondering if the mock should be in the individual test or in the [jest tests setup](https://github.com/artsy/reaction/blob/4c6d2e125dd32a90ee780a14d2f207c847defc43/src/setup_jest.ts#L29-L35). I'm more so wondering when should it be in the individual test vs the the test setup for my own knowledge.

Reply:
That seems like a philosophical question 🙂 Here's my take: It makes sense to put a mock in the global setup file if it needs to be mocked in a particular way most of the time. Good examples are libs that perform network requests like `react-tracking`. Mocking too many things might lead to surprising behaviour for people writing new tests or refactoring old tests. That kind of thing can cost people a lot of time and frustration in debugging. Explicit local mocking can also make tests easier to read than implicit global mocking.

In other words... 🤷‍♂️:D